tutorials/text/text_props.py

```diff
@@ -204,14 +212,20 @@
 # Korean, or Japanese.
 #
 # To set the default font to be one that supports the code points you
-# need, prepend the font name to ``'font.family'`` or the desired alias
-# lists ::
+# need, prepend the font name to ``'font.family'`` or to the desired alias
+# lists, latter of which is not recommended.
+#
+# This is because these generic family alias lists should ideally contain fonts
+# that are shipped alongside Matplotlib, which work across all installations,
+# unlike user-defined fonts which have no such guarantee. ::
 #
-#    matplotlib.rcParams['font.sans-serif'] = ['Source Han Sans TW', 'sans-serif']
+#    # A generic-family is encouraged to be appended to the font-family list
```

Comment:
appended, or prepended?

Reply:
Appended, we're talking about generic family here (sans-serif in this case).

Reply:
Ah, I misread that.